gnome-base/gnome-desktop/gnome-desktop-43-r400.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome.org meson xdg
+
+DESCRIPTION="Library with common API for various GNOME modules"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-desktop/"
+
+LICENSE="GPL-2+ LGPL-2+ FDL-1.1+"
+SLOT="4/1" # subslot = libgnome-desktop-4 soname version
+IUSE="debug gtk-doc seccomp systemd udev introspection"
```

Comment:
`IUSE` should always be alphabetical, but... I intentionally dropped the `introspection` USE flag from the -r400 ebuild in

````
commit baf18a93c451c680657b01e7483027fb853cc090
Author: Matt Turner <mattst88@gentoo.org>
Date:   Mon Jul 11 22:59:18 2022 -0700

    gnome-base/gnome-desktop: Version bump to 42.3
    
    * Drop IUSE=introspection from -r400 (bug #856649)
    
    Closes: https://bugs.gentoo.org/856649
    Signed-off-by: Matt Turner <mattst88@gentoo.org>
````

I don't believe it's possible to disable introspection in -r400, and with `USE=-introspection` the build fails exactly the same way as it did as reported in that bug.